EIPS/eip-draft_hfmeta_eip150.md

```diff
@@ -0,0 +1,26 @@
+## Preamble
+
+    EIP: <to be assigned>
+    Title: Hardfork Meta: EIP 150 Hard Fork
+    Author: Alex Beregszaszi
+    Type: Standard Track
+    Category: Core
+    Status: Final
+    Created: 23/04/2017
+    Requires: 150
+
+## Abstract
+
+This specifies the changes included in the hard fork named EIP 150 Hard Fork.
+
+## Specification
+
+- Codename: EIP 150 Hard Fork
```

Comment:
The codename for the EIP-150 fork was "Tangerine Whistle" (mentioned in the [blog post](https://blog.ethereum.org/2016/11/18/hard-fork-no-4-spurious-dragon/) on Spurious Dragon).